test/e2e/gardener/shoot/internal/node/node.go

```diff
@@ -66,22 +66,22 @@ func VerifyNodeCriticalComponentsBootstrapping(ctx context.Context, f *framework
 
 	By("Delete Nodes and Machines to trigger new Node bootstrap")
 	Eventually(func(g Gomega) {
```

Comment:
Shouldn't it be rather `EventuallyWithOffSet` and `ConsistentlyWithOffset` instead of offsetting the expectations inside? same below